dashboard/app/helpers/openai_chat_helper.rb

```diff
@@ -0,0 +1,37 @@
+module OpenaiChatHelper
```

Comment:
I am just curious if this helper has been extracted to keep logic out of the controller, or if you envision reusing this helper in another controller?

Reply:
I extracted these these to make it easier to stub in the unit tests - similar to the helper `javalab_files_helper.rb` whose functions are stubbed in `javabuilder_sessions_controller_test.rb`. Addressed this question [here](https://github.com/code-dot-org/code-dot-org/pull/52759#discussion_r1269991942).